src/CustomMarshalers/src/Configurations.props

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <PropertyGroup>
+    <BuildConfigurations>
+      netcoreapp-Windows_NT;
```

Comment:
You're saying this is inbox in .netcore3.0 but it doesn't have a unix implementation... we should at least create a PNSE assembly for Unix if we want these to go inbox.

cc: @ericstj

Reply:
If it is all going to be just an internal API in CoreLib to support Windows-specific COM interop, it does not need to have PNSE assembly on Unix.

Reply:
Yes, agree. I just didn't know if we were planning on exposing this guy.